lib/galaxy_test/api/test_libraries.py

```diff
@@ -427,10 +428,13 @@ def test_create_dataset_in_subfolder(self):
         create_response = self._post(f"folders/{subfolder_id}/contents", payload, json=True)
         self._assert_status_code_is(create_response, 200)
         self._assert_has_keys(create_response.json(), "name", "id")
-        dataset_update_time = create_response.json()["update_time"]
+        dataset_update_time = datetime.fromisoformat(create_response.json()["update_time"])
         container_fetch_response = self.galaxy_interactor.get(f"folders/{folder_id}/contents")
-        container_update_time = container_fetch_response.json()["folder_contents"][0]["update_time"]
-        assert dataset_update_time == container_update_time, container_fetch_response
+        container_update_time = datetime.fromisoformat(
+            container_fetch_response.json()["folder_contents"][0]["update_time"]
+        )
+        # Assert that the update time of the dataset is approximately the same as the update time of the container
+        assert abs((dataset_update_time - container_update_time).total_seconds()) < 1
```

Comment:
Wouldn't be better to just check that the subfolder contains the dataset, instead of looking at update times?

Reply:
Yep, that would make more sense, but since the test is 6 years old I didn't know if there was a good reason for checking this so... :sweat_smile: 

I will drop this check and just assert the dataset is there :+1: